CHANGELOG.md

```diff
@@ -27,6 +27,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -->
 
+## [Unreleased]
+
+### Changed 🛠
+- [PR#990](https://github.com/EmbarkStudios/rust-gpu/pull/990) Removed return type inference from `Image` API and made `glam` usage mandatory.
```

Comment:
Should we mark this explicitly as backwards-incompatible?
Also, should we remove the `optional = true` from here?
https://github.com/EmbarkStudios/rust-gpu/blob/fd4da169e41fc42ed3a9ede926a3116dddb7019d/crates/spirv-std/Cargo.toml#L15

Reply:
> Should we mark this explicitly as backwards-incompatible?
Not sure what you mean?

> Also, should we remove the optional = true from here?
Yeah I thought about that. The reason I kept it as non-default is that I don't want to silently force glam onto people that are currently NOT using glam. Especially if we want to add support for some other vector library, you'd need to consciously make the choice which one you want. Do you agree with this reasoning?